pkg/providers/amifamily/al2.go

```diff
@@ -80,6 +64,30 @@ func (a AL2) DescribeImageQuery(ctx context.Context, ssmProvider ssm.Provider, k
 	}, nil
 }
 
+func (a AL2) SSMParameters(k8sVersion string, amiVersion string) []string {
```

Comment:
If this is part of the interface, I suspect that the SSM provider can probably just call it